jerry-math/jerry-math-internal.h

```diff
@@ -35,7 +35,7 @@
  * If byte order is supported and set to little or target is among common
  * cases checked define __LITTLE_ENDIAN.
  */
-#if (defined (i386) || defined (__i386) || defined (__i386__) || \
+#if (defined (_MSC_VER) || defined (i386) || defined (__i386) || defined (__i386__) || \
```

Comment:
This is not correct. MSVC can compile for multiple architectures, e.g., for ARM (bot 32 and 64 bit). The macros to be tested for are `_M_IX86` and `_M_X64` (see: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-160 )

Moreover, add these tests on a new line after `defined (i86pc)`.

Reply:
I've guard _MSC_VER with separate condition, this line are too long and also gcc specfic